### PR TITLE
Translate rework

### PR DESCRIPTION
**Pull request by Wax6610**

### CODE REVIEW

File: components/heroes/TalentSection.tsx [hunk 1]
```diff
@@ -62,48 +69,53 @@ export function TalentSection({
...
-            <li>Light: Level 10 Intimacy + {bondRequirments.bond2}</li>
-            <li>Honor: Level 15 Intimacy + {bondRequirments.bond3}</li>
-            <li>Toughness: Level 23 Intimacy + {bondRequirments.bond4}</li>
-            <li>Strength: Level 25 Intimacy + {bondRequirments.bond5}</li>
+            <p className="pt-5 font-bold">{t('Bond Requirement')}:</p>
+            <li>{`${t('Glory')}: ${t('Level')} 5 ${t('Intimacy')}`}</li>
+            <li>{`${t('Light')}: ${t('Level')} 10 ${t('Intimacy')} + ${bond2}`}</li>
+            <li>{`${t('Honor')}: ${t('Level')} 15 ${t('Intimacy')} + ${bond3}`}</li>
+            <li>{`${t('Toughness')}: ${t('Level')} 23 ${t('Intimacy')} + ${bond4}`}</li>
+            <li>{`${t('Strength')}: ${t('Level')} 25 ${t('Intimacy')} + ${bond5}`}</li>
```
Comment (RedWrath5) [inline components/heroes/TalentSection.tsx:77]: incorrect levels

File: components/soldiers/SoldierGallerySection.tsx [hunk 2]
```diff
@@ -21,29 +27,31 @@ export function SoldiersGallerySection({ soldier }: { soldier: Soldier }) {
...
-              className={
-                showMore
-                  ? "transform -rotate-180 bg-white -mt-6"
-                  : "bg-white -mt-6"
-              }
-            />
+            {
+              <ArrowDropDownCircleOutlined
+                className="text-4xl -mt-6 bg-white"
+                //tailwind classes not working here. mui overwrite it
```
Comment (RedWrath5) [inline components/soldiers/SoldierGallerySection.tsx:47]: MUI has a fontSize prop, did that not work?

https://mui.com/api/icon/

File: pages/equipment.tsx [hunk 3]
```diff
@@ -3,22 +3,42 @@ import { Equipment } from "../types/hero";
...
+  translateUIMap: TranslateUILanguageMap;
+  translateEquipmentMap: TranslateEquipmentLanguageMap;
+}>) => (
+  <TranslateWrapper translateMap={translateUIMap}>
+    <Layout>
+      <EquipmentTranslateWrapper translateMap={translateEquipmentMap}>
+        <EquipmentPage equipment={equipment} />
+      </EquipmentTranslateWrapper>
+    </Layout>
+  </TranslateWrapper>
```
Comment (RedWrath5) [inline pages/equipment.tsx:28]: this is alittle weird, but we can fix after

File: pages/heroes/[name].tsx [hunk 4]
```diff
@@ -40,15 +119,34 @@ export const getStaticPaths: GetStaticPaths = async () => {
...
+      skillsMap,
+      soldierMap,
+      translateUIMap,
       skillsToHeroMap,
+      translateHeroMap,
+      translateClassMap,
+      threeCostSkillMap,
+      translateSkillsMap,
+      translateSoldiersMap,
+      translateEquipmentMap,
```
Comment (RedWrath5) [inline pages/heroes/[name].tsx:149]: I see why you got forced into this. I will think of solutions post merge

**Review by RedWrath5 — APPROVED** [review]

File: components/Layout.tsx [hunk 5]
```diff
@@ -7,7 +7,7 @@ export function Layout({ children }: PropsWithChildren<{}>) {
       <div id="page-container" className="flex flex-col justify-between">
         <div id="content-wrap">
           <Header />
-          <main className="flex pt-14 sm:pt-0">{children}</main>
+          <main className="flex pt-14 sm:pt-0">{children}</main>=
```
[the remark this answers is not in the document]
Comment (Wax6610) [reply components/Layout.tsx]: fixed

File: components/heroes/TalentSection.tsx (hunk 1 above)
Comment (Wax6610) [reply components/heroes/TalentSection.tsx]: fixed

File: components/soldiers/SoldierGallerySection.tsx (hunk 2 above)
Comment (Wax6610) [reply components/soldiers/SoldierGallerySection.tsx]: fixed

File: pages/heroes/[name].tsx (hunk 4 above)
Comment (Wax6610) [reply pages/heroes/[name].tsx:149]: yeah. Hero page require too much different data. Perhaps i should transform all related data into super object. Just for this page.

**Pull request closed without merging**